jupyter_console/ptshell.py

```diff
@@ -463,6 +472,16 @@ def pre_prompt(self):
             b.cursor_position += b.document.get_end_of_document_position()
 
     def interact(self, display_banner=None):
+        # Start handle_iopub thread for external input
+        def external_monitor():
+            while self.keep_running:
+                self.handle_iopub()
```

Comment:
Looks like my other commit got into this. Sorry. The lines here are specifically for #105.